lms/djangoapps/grades/tasks.py

```diff
@@ -69,7 +69,7 @@ def compute_all_grades_for_course(**kwargs):
                 'batch_size': batch_size,
             })
             compute_grades_for_course_v2.apply_async(
-                kwargs=kwargs, routing_key=settings.POLICY_CHANGE_GRADES_ROUTING_KEY
+                kwargs=kwargs, queue=settings.POLICY_CHANGE_GRADES_ROUTING_KEY
```

Comment:
It looks like there's still a use of the `routing_key` parameter for this task in https://github.com/edx/edx-platform/blob/master/lms/djangoapps/grades/management/commands/compute_grades.py#L83 .